File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_7.go
package main
import ("fmt"
        
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    var c float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&c)
    var d float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&d)
    if (a == b) && (b == c) && (a == c) {
        fmt.Print ("Nomer chisla, otlichnogo ot ostalnih: 4")
    } else if (a == c) && (c == d) && (a == d) {
        fmt.Print ("Nomer chisla, otlichnogo ot ostalnih: 2")
    }  else if (a == b) && (b == d) && (a == d) {
        fmt.Print ("Nomer chisla, otlichnogo ot ostalnih: 3")
    } else if (b == c) && (c == d) && (b == d) {
        fmt.Print ("Nomer chisla, otlichnogo ot ostalnih: 1")
    } else {
        fmt.Print ("Nekorrektniyi vvod")
    }
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_7.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
c = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
d = gets.to_i
if (a == b) & (b == c) & (a == c)
    puts ('Nomer chisla, otlichnogo ot ostalnih: 4')
elsif (a == c) & (c == d) & (a == d)
    puts ('Nomer chisla, otlichnogo ot ostalnih: 2')
elsif (a == b) & (b == d) & (a == d)
    puts ('Nomer chisla, otlichnogo ot ostalnih: 3')
elsif (b == c) & (c == d) & (b == d)
    puts ('Nomer chisla, otlichnogo ot ostalnih: 1')
else
    puts ('Necorrectniyi vvod')
end